Lexplorer/Lexplorer.csproj

```diff
@@ -27,6 +27,12 @@
     <PackageReference Include="Blazor-ApexCharts" Version="0.9.7-beta" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Update="appsettings.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```

Comment:
Didn't see that mentioned anywhere - what problem did this fix? I never experienced issues with out of date appsettings.

Reply:
thats a bad habit of mine from previous projects. fixed in [c013e96](https://github.com/fudgebucket27/Lexplorer/pull/182/commits/c013e96cdf9ed1e8675463fce642052181076d8a)

Reply:
Originally, the whole ItemGroup wasn't there. Now its changed to PreserveNewest, which might have been the default, but I don't know.

Reply:
yeh it was definitely copy if newer before.